scripts/linux-prereqs.sh

```diff
@@ -43,7 +43,7 @@ elif type yum  > /dev/null 2>&1; then
 elif type pacman > /dev/null 2>&1; then
     echo "(*) Detected ArchLinux (unoffically/community supported)"
     echo ""
-    sudo pacman -Sq libunwind lttng-ust curl openssl-1.0 libutil-linux krb5 icu zlib gettext desktop-file-utils gnome-keyring libsecret
+    sudo pacman -Sq --needed grc liburcu libunwind lttng-ust curl openssl-1.0 libutil-linux krb5 icu zlib gettext desktop-file-utils gnome-keyring libsecret
```

Comment:
I am getting a "target not found" error for "grc".  Should that be gcc?

Reply:
Oh just a typo, sorry I met `gcr`